cookbooks/aws-parallelcluster-slurm/resources/munge/partial/_munge_actions.rb

```diff
@@ -25,6 +28,21 @@
 munge_group_id = node['cluster']['munge']['group_id']
 
 action :setup do
+  directory node['cluster']['sources_dir'] do
```

Comment:
why are we creating sources_dir here? We should rely on directories.rb and set it as dep.